kaldb/src/main/java/com/slack/kaldb/logstore/search/LogIndexSearcherImpl.java

```diff
@@ -111,30 +129,39 @@ public SearchResult<LogMessage> search(
       // This is a useful optimization for indexes that are static.
       IndexSearcher searcher = searcherManager.acquire();
       try {
-        TopFieldCollector topFieldCollector = buildTopFieldCollector(howMany);
-        StatsCollector statsCollector =
+        CollectorManager<TopFieldCollector, TopFieldDocs> topFieldCollector =
```

Comment:
I think the code is easier to read if this code is moved into the conditional below.

Reply:
Done.